### DISCUSSION TITLE
Support for traveling outside of starting zone.

### DISCUSSION THREAD

**Discussion by jamiemaertens** [ehmorris/lunar-lander#30] [Ideas]
This could be done by moving the camera or potentially showing the ship in a little bubble with it's current orientation. I prefer moving the camera since the playing space would be bigger.

**Comment by ehmorris** [comment]
Thanks for the suggestion!

**Comment by ehmorris** [comment]
I sorta implemented this. The ship now appears inside a bubble when you leave the upper bounds of the play space. Horizontal motion is the same however.

**Comment by ehmorris** [comment]
Another update! You can now fly upwards and instead of a bubble, you enter a new game mode where you have to dodge asteroids
